client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -52,12 +60,20 @@
 	flex: auto;
 	overflow-y: auto;
 
+	@include breakpoint( ">480px" ) {
+		padding: 24px;
+	}
+
 	@include breakpoint( ">660px" ) {
 		display: flex;
+
+		.upload-image {
+			margin-right: 25px;
+		}
 	}
 
 	.upload-image {
-		margin-right: 25px;
+		margin-bottom: 20px;
```

Comment:
The reason I pick 20px was all `..form-fieldset` has bottom margin of 20px. I'm not sure why it isn't 24px, but that also need some research.